src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -124,6 +124,19 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHABF16_4D,
                                             ::testing::Values(CPUTestUtils::empty_plugin_config)),
                          MHA::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHAFP16_4D,
```

Comment:
This test creates full fp16 model with f16 MHA. I think we should cover the case with `--infer_prcision=f16` for f32 model by tests too.
If the plugin supports `--infer_precision=fp16`, could you please the test `smoke_Snippets_MHAEnforceFP16` as it's done for bf16 below?

Reply:
done, thanks Alexandra!